dxfeed/wrappers/DXFeedSubscription.py

```diff
@@ -0,0 +1,82 @@
+from dxfeed.core.DXFeedPy import *
+from typing import Iterable, Union
+from datetime import datetime
+import pandas as pd
+from warnings import warn
+
+
+class DXFeedSubscription(object):
+    def __init__(self, connection, event_type: str, date_time: Union[str, datetime], data_len: int = 100000):
+        if date_time is not None:
+            if isinstance(date_time, str):
+                try:
+                    date_time = datetime.strptime(date_time, '%Y-%m-%d %H:%M:%S.%f')
+                except ValueError:
+                    try:
+                        date_time = pd.to_datetime(date_time, format='%Y-%m-%d %H:%M:%S.%f', infer_datetime_format=True)
+                        warn_message = 'date_time argument does not exactly match %Y-%m-%d %H:%M:%S.%f format,' + \
+                                       ' date was parsed automatically as ' + \
+                                       date_time.strftime(format="%Y-%m-%d %H:%M:%S.%f")
+                        warn(warn_message)
+                    except ValueError:
+                        raise ValueError('date_time should use %Y-%m-%d %H:%M:%S.%f format!')
+            timestamp = int(date_time.timestamp() * 1000)
+            self.__sub = dxf_create_subscription_timed(cc=connection,
+                                                       event_type=event_type,
+                                                       time=timestamp,
+                                                       data_len=data_len)
+        else:
+            self.__sub = dxf_create_subscription(cc=connection,
+                                                 event_type=event_type,
+                                                 data_len=data_len)
+
+    @property
+    def event_type(self):
+        return dxf_get_subscription_event_types(self.__sub, return_str=True)
+
+    @property
+    def symbols(self):
+        return dxf_get_symbols(self.__sub)
+
+    def add_symbols(self, symbols: Iterable[str]):
+        dxf_add_symbols(sc=self.__sub, symbols=symbols)
+        return self
+
+    def remove_symbols(self, symbols: Iterable[str] = [''], remove_all: bool = False):
+        if remove_all:
+            dxf_clear_symbols(self.__sub)
+        else:
+            dxf_remove_symbols(self.__sub, symbols=symbols)
+        return self
+
+    def close_subscription(self):
+        dxf_close_subscription(sc=self.__sub)
```

Comment:
Mb we can also detach all listeners, so user wouldn't be forced to do it in a manual way?

Reply:
I don't quite understand: you offer to add `detach_listener` to `close_subscription`? If yes, there is no need for it. C-API deals with everything related to the subscription on its side.